public/stylesheets/userProfile.css

```diff
@@ -28,6 +28,103 @@ svg {
     border-top: 0;
 }
 
+#profile-container {
+    margin: 0px;
+    margin-top: 15px;
+}
+
+#org-container {
+    display: flex;
+}
+
+#org-name-container {
+    display: flex;
+    font-size: 30px;
+    font-family: raleway, sans-serif;
+    margin-right: 15px;
+}
+
+#edit-org-dropdown {
+    display: flex;
+    align-items: center;
+}
+
+#edit-org-button {
+    text-align: center;
+    padding: 5px 25px;
+    cursor: pointer;
+}
+
+#edit-org-menu {
+    cursor: pointer;
+    z-index: 2000;
+    top: 90%;
+}
+
+#edit-org-info {
+    bottom: 0px;
+    left: 105%;
+}
+
+.username {
+    font-size: 40px;
+    font-family: raleway-bold, sans-serif;
+}
+
+.org-content {
+    border-radius: 10px;
+    border: 1px solid #2D2A3F;
+    color: #2D2A3F;
+    background-color: white;
+    font: normal normal normal 14px/1.4em raleway, sans-serif;
+    font-family: 'Raleway', sans-serif;
```

Comment:
I think the `font-family` and `font-size` properties are both covered by the `font` property already.